pubspec.yaml

```diff
@@ -15,6 +15,8 @@ dependencies:
 dev_dependencies:
   build_runner: 1.0.0
   build_web_compilers: any
+  flutter_tools:
```

Comment:
lets see how many files we need to duplicate to avoid this dependency.
worst case we could enforce a project structure of devtools with tip of trunk flutter side by side.

Reply:
Added dep files.